### ISSUE TITLE
Lowest price on Allegro via API

### ISSUE THREAD

**Issue by dominikahr** [allegro/allegro-api#9419]
Hi there, 

I just got an emil that sellers now have the possibility ti use the "Lowest price on Allegro" functionality. I was wondering how does it work via API? I did not find any instructions on how to implemement this.

Thanks in advance! 

All the best, 

Dominika

**Comment by Lukasz-Zurek** [comment]
Hello!

I understand that you meant [this article](https://help.allegro.com/sell/en/a/we-have-enabled-the-new-pricing-rule-check-how-lowest-price-on-allegro-works-RdR0YV3v2fj). We do not have it on Allegro API. We only support one type of price rules:

```
{
    "rules": [
        {
            "id": "646364e007bd9c00dd95fc52",
            "type": "EXCHANGE_RATE",
            "updatedAt": "2023-05-16T11:11:28Z"
        }
    ]
}
```

which converts prices in offers on additional marketplaces according to the current exchange rate.

**Comment by dominikahr** [comment]
@Lukasz-Zurek And can we at least somehow find out, that the seller has this new pricing rule activated for some offer so that we would not update the price for it via API?

**Comment by Lukasz-Zurek** [comment]
@dominikahr No. But I can assure you that we are planing to introduce it un our API pretty soon and we will inform about this change in [our news](https://developer.allegro.pl/news).

**Comment by dominikahr** [comment]
Ok, thank you for the info :)
